task-sdk/src/airflow/sdk/definitions/baseoperatorlink.py

```diff
@@ -38,6 +37,8 @@ class XComOperatorLink(LoggingMixin):
     xcom_key: str
 
     def get_link(self, operator: BaseOperator, *, ti_key: TaskInstanceKey) -> str:
+        from airflow.sdk.execution_time.xcom import BaseXCom
```

Comment:
We shouldn't be using this as "BaseXCom" really -- that feels like an odd interface. I think the we should do the "export it as XCom" that we did before.

Reply:
Yeah, there's an `XCom` exported from that 
```suggestion
        from airflow.sdk.execution_time.xcom import XCom
```

Reply:
And then change L52 to `XCom.get()` I think?

Reply:
Makes sense. Its get_one itself though

Reply:
Handled in dc8294832fe41ea7f9b4ad4628f7db841b9db063